bagit.xml

```diff
@@ -202,12 +202,15 @@ transfer.
 <section title="Purpose">
 <t>
 BagIt is a hierarchical file packaging format designed to support
-disk-based or network-based storage and transfer of arbitrary digital
-content. A bag consists of a "payload" and "tags". The content of the payload
-is the custodial focus of the bag and is treated as semantically opaque.
-The "tags" are metadata files intended to facilitate and document the storage
-and transfer of the bag. The name, BagIt, is inspired by the "enclose and deposit" method
-<xref target="ENCDEP" />, sometimes referred to as "bag it and tag it".
+storage and transfer of arbitrary digital content (i.e. files). 
+A bag consists of a directory containing the payload files and other accompanying
+files known as "tag" files. The "tags" are metadata files intended to facilitate 
+and document the storage and transfer of the bag. 
+BagIt does not require the processing of the entire structure to understand to
+involve understanding the payload. The name, BagIt, is inspired by the "enclose 
+and deposit" method <xref target="ENCDEP" />, sometimes referred to as "bag it and tag it".
+This differs from other specifications (like ZIP) that only ensure the integrity
```

Comment:
This isn't really true for ZIP files – by now that format includes integrity checks on the entire archive as well. The differences I would emphasize are the use of stronger checks than CRC-32s, ability to upgrade to stronger hash algorithms without breaking backwards compatibility, etc. and the tradeoffs for large archives of having each file be individually accessible, transferred, etc. without having to read through a potentially large archive (e.g. people still report interoperability issues with archives which are large enough to require using ZIP64.